docs/deploy_route53.md

```diff
@@ -71,7 +71,7 @@ kubectl -n test-gslb get gslb test-gslb-failover -o yaml
 aws route53 list-resource-record-sets --hosted-zone-id $YOUR_HOSTED_ZONE_ID
 ```
 
-You should see that `gslb-ns-$dnsZone-$geotag` NS and glue A records were created to
+You should see that `gslb-ns-$geotag-flatten($dnsZone)` NS and glue A records were created to
```

Comment:
it's not obvious what flatten means here without the context or example

Reply:
Extending with example:

_You should see new NS and glue A records were created to
automatically configure DNS zone delegation._

_The NS record is calculated from $dnsZone, $edgeDnsZone and $geotag.
For example: `gslb-ns-us-test-gslb.cloud.example.com` will be calculated from 
following values:_
```
$geotag = cloud.example.com
$dnsZone = test.gslb.cloud.example.com
$edgeDnsZone = cloud.example.com
```

Reply:
Sounds good!